<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Owner_Interface</class>
 <widget class="QWidget" name="Owner_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Welcome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome to application,</string>
   </property>
  </widget>
  <widget class="QLabel" name="Owner_FirstName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="owner_prof_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Place_cargo_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Place new order!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="History_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Check order history</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Order_status">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Order</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
